src/Postmark/Model/Suppressions/PostmarkSuppressionOrigin.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Postmark.Model.Suppressions
+{
+    /// <summary>
+    /// Origin that generated a Suppression
+    /// </summary>
+    public enum PostmarkSuppressionOrigin
+    {
+        Customer, SpamComplaintAuthorizedCustomer, Recipient, Admin
```

Comment:
Will a reactivation record ever be returned from the API? Does the spam complaint authorized customer name make any sense to our users?

Reply:
You're right. `SpamComplaintAuthorizedCustomer` is only used for reactivations and that's never returned from the API. Those were updated to `strings` now anyway, but good point.